src/Umbraco.Web.BackOffice/Install/InstallAreaRoutes.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Umbraco.Cms.Web.BackOffice.Install;
 
+[Obsolete("Has been replaced with attribute routing in the new backoffice API")]
```

Comment:
The obsolete message should not be past tense - maybe "Will be replaced with attribute routing in the new backoffice API" instead?